debian/debian/ix-vendor.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+After=middlewared.service
+
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+TimeoutStartSec=1200
```

Comment:
I think this should be smaller so it does not stop the boot process if something goes south and also probably it will never take this long?

Reply:
This was just copied from another unrelated service file so I'll look into this too.